app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/courselist/view/CourseListFragment.kt

```diff
@@ -105,7 +106,11 @@ class CourseListFragment : Fragment() {
     fun showMessage() {
         courselistViewModel.showMsg.observe(this, androidx.lifecycle.Observer {
             val message = it?.showMsg
-            Toast.makeText(context, message, Toast.LENGTH_LONG).show()
+            view?.let { it1 ->
+                if (message != null) {
+                    Snackbar.make(it1, message, Snackbar.LENGTH_LONG).show()
```

Comment:
Use existing utils method
'view?.showSnackbar(message)'
 instead of 
Snackbar.make(it1, message, Snackbar.LENGTH_LONG).show()

Add an extra parameter in ViewUtils if you need.